.github/workflows/parallel-tests.yaml

```diff
@@ -4,6 +4,12 @@
 name: Helper workflow for E2E Tests
 on:
   workflow_call: # This is the event that triggers the workflow
+    inputs:
+        fetch_zarf:
```

Comment:
What are the teams thoughts about making this input (or likely adding an additional input) a ref string?

The nightly cron can trigger with the ref of `main` but we could trigger a dispatch with a specific branch/ref if there's a specific branch we're interested in testing compatibility against.

Reply:
I would support this enhancement but we can also do it ourselves later